Example/WalletConnect/WCSessionViewController.swift

```diff
@@ -129,7 +152,7 @@ class ViewController: UIViewController {
 
         var result = privateKey.sign(digest: Hash.keccak256(data: data), curve: .secp256k1)!
         result[64] += 27
-        self.interactor?.approveRequest(id: id, result: result.hexString).cauterize()
+        self.interactor?.approveRequest(id: id, result: "0x" + result.hexString).cauterize()
```

Comment:
did you consider adding 0x prefix extension?

Reply:
`result.hexString.0xPrefix` or something similar?  `result.0xhex`

Reply:
we can add that, but it's an example app, worth to do it?